docs/book/src/reference/creating-events.md

```diff
@@ -23,6 +23,8 @@ Event(object runtime.Object, eventtype, reason, message string)
 - `eventtype` is this event type, and is either *Normal* or *Warning*.
 - `reason` is the reason this event is generated. It should be short and unique with `UpperCamelCase` format. The value could appear in *switch* statements by automation.
 - `message` is intended to be consumed by humans.
+More [info](https://github.com/kubernetes/api/blob/6c11c9e4685cc62e4ddc8d4aaa824c46150c9148/core/v1/types.go#L6019-L6024):
```

Comment:
@Sajiyah-Salat see that each spec in the Kubernertes Api Event is described and point out by varsha in the previous suggestion comment.

Please, ensure that you click on in each link to check them out and add the [[more info][link]] for each case scenario.

Also, remember that ALL links should be added as alias and the alias should defined at the bottom.

By last, do not forget to squash all commits and that you addressed all comments and suggestions